admin/src/components/Item/ItemCardHeader/index.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+//@ts-ignore
+import { Flex } from '@strapi/design-system/Flex';
+//@ts-ignore
+import { Typography } from '@strapi/design-system/Typography';
+//@ts-ignore
+import { IconButton } from '@strapi/design-system/IconButton';
+//@ts-ignore
+import { Icon } from '@strapi/design-system/Icon';
+//@ts-ignore
+import DragButton from '../../DragButton';
+import Wrapper from './Wrapper';
+import ItemCardBadge from '../ItemCardBadge';
+import { getMessage } from '../../../utils';
+import { ToBeFixed, VoidEffect } from '../../../../../types';
+import { pencilIcon, refreshIcon, trashIcon } from './icons';
+
+interface IProps {
+  title: string,
+  path: string,
+  icon: ToBeFixed,
+  removed: boolean,
+  onItemRemove: VoidEffect,
+  onItemEdit: VoidEffect,
+  onItemRestore: VoidEffect,
+  dragRef: React.MutableRefObject<HTMLHeadingElement>,
+}
```

Comment:
move to separate file with types

Reply:
I think it can be omitted here since there is only one type in this file.